.gitignore

```diff
@@ -691,7 +673,12 @@ FodyWeavers.xsd
 # Old /mp directory should no longer be used
 /mp
 
-/game/neo/cache/DownloadCache.db
-/game/neo/lights.rad
+# This is source-controlled now, but it exists in the "bin" dir, so we ignore this old location
 /game/neo/rebuild.fgd
-/game/neo/detail.vbsp
+
+/game/neo/cache/DownloadCache.db
+
+!/game/bin/
+!/game/bin/x64/
+!/game/bin/**/rebuild.fgd
+!/game/bin/[Gg]ame[Cc]onfig.txt
```

Comment:
Sorry, I typo'd the syntax here on lines 683-684. We'd instead want the exclusions to be:

```diff
diff --git a/.gitignore b/.gitignore
index 51bd5ebe..eb1317a8 100644
--- a/.gitignore
+++ b/.gitignore
@@ -680,5 +680,5 @@ FodyWeavers.xsd

 !/game/bin/
 !/game/bin/x64/
-!/game/bin/**/rebuild.fgd
-!/game/bin/[Gg]ame[Cc]onfig.txt
+!/game/bin/rebuild.fgd
+!/game/bin/**/[Gg]ame[Cc]onfig.txt
```

so the double-asterisk `/**/` token from line 683 should go with the GameConfig entry at line 684 (since we wanna match both the base "bin" dir and the "x64" dir GameConfigs).

And for the FGD, we only want to match the base "bin" dir entry, since the FGD is not supposed to exist in the "x64" dir at all.